docs/rules/yoda.md

```diff
@@ -127,6 +127,20 @@ function howLong(arr) {
 }
 ```
 
+### Apply only to equality, but not other operators
```

Comment:
I know this md file didn't have options section before. But now that we added an extra option, I think we should reformat it to have options section (like other rules) and show examples. Right now it's not clear whether I can use both `onlyEquality` and `ranges` options, or if I only want to use `onlyEquality` option, should it be third array item or second.